client/my-sites/plans/current-plan/ecommerce-trial/index.jsx

```diff
@@ -0,0 +1,242 @@
+			showButton: true,
+			buttonText: translate( 'Ask a question' ),
+		},
+		{
+			illustration: '/calypso/images/plans/wpcom/ecommerce-trial/premium-themes.svg',
+			title: translate( 'Premium themes' ),
+			text: translate( 'Explore a diverse selection of beautifully designed premium themes.' ),
+			showButton: true,
+			buttonText: translate( 'Browse premium themes' ),
+		},
+		{
+			illustration: '/calypso/images/plans/wpcom/ecommerce-trial/advanced-customization.svg',
+			title: translate( 'Advanced customization' ),
+			text: translate( "Change your store's appearance in a few clicks!" ),
+			showButton: true,
+			buttonText: translate( 'Design your store' ),
+		},
+		{
+			illustration: '/calypso/images/plans/wpcom/ecommerce-trial/unlimited-products.svg',
+			title: translate( 'Unlimited products' ),
+			text: translate( 'List as many products or services as you’d like and offer subscriptions.' ),
+			showButton: true,
+			buttonText: translate( 'Add a product' ),
+		},
+		{
+			illustration: '/calypso/images/plans/wpcom/ecommerce-trial/jetpack-features.svg',
+			title: translate( 'Jetpack features' ),
+			text: translate( 'Get auto real-time backups, malware scans, and spam protection.' ),
+			showButton: true,
+			buttonText: translate( 'Keep your store safe' ),
+		},
+		{
+			illustration: '/calypso/images/plans/wpcom/ecommerce-trial/seo-tools.svg',
+			title: translate( 'SEO tools' ),
+			text: translate(
+				'Boost traffic with tools that make your content more findable on search engines.'
+			),
+			showButton: true,
+			buttonText: translate( 'Increase visibility' ),
+		},
+		{
+			illustration: '/calypso/images/plans/wpcom/ecommerce-trial/google-analytics.svg',
+			title: translate( 'Google Analytics' ),
+			text: translate(
+				'Understand visitors and traffic patterns more in depht with Google stats.'
+			),
+			showButton: true,
+			buttonText: translate( 'Connect Google Analytics' ),
+		},
+		{
+			illustration: '/calypso/images/plans/wpcom/ecommerce-trial/best-in-class-hosting.svg',
+			title: translate( 'Best-in-class hosting' ),
+			text: translate(
+				'Hosting is included with your plan, eliminating additional cost and technical hassle.'
+			),
+			showButton: false,
+		},
+	];
+
+	const whatsIncluded = displayAllIncluded
+		? allIncludedFeatures
+		: // Show only first 4 items
+		  allIncludedFeatures.slice( 0, 4 );
+
+	const whatsNotIncluded = [
+		{
+			illustration: '/calypso/images/plans/wpcom/ecommerce-trial/launch.png',
+			title: translate( 'Launch your store to the world' ),
+			text: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis quis magna ac odio ullamcorper efficitur.',
+		},
+		{
+			illustration: '/calypso/images/plans/wpcom/ecommerce-trial/themes.png',
+			title: translate( 'Access all premium themes' ),
+			text: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis quis magna ac odio ullamcorper efficitur.',
+		},
```

Comment:
Users on the free trial already have access to first-party premium themes, so this should be de-prioritised for now, especially relative to the other features.